### DISCUSSION TITLE
Metaviral SPAdes intermediate results

### DISCUSSION THREAD

**Discussion by eloilit** [ablab/spades#737] [Q&A] [answered]
Hi !

I'm currently trying to use metaviral SPAdes and would like to access the intermediate results, that is, the results yielded after viralAssembly and before the viralVerify check. Are those intermediate results the ones in "before_rr.fasta" ?

Best,

Eloi

**Comment by asl** [comment, marked as answer]
metaviralSPAdes does not run viralVerify. Output is 'contigs.fasta' as specified in https://cab.spbu.ru/files/release3.15.2/manual.html#spadesoutsec
